Sources/App/Views/Home/HomeIndex+View.swift

```diff
@@ -41,6 +41,7 @@ enum HomeIndex {
                     .div(
                         .class("inner"),
                         .h3("The place to find Swift packages."),
+                        .structuredData(IndexSchema()),
```

Comment:
Not sold on the location of this, open to suggestions. The structured data is technically specific to the homepage, but it's only relevant to the search box (currently) which is why I put it here.

Could just as easily make sense right at the top.

Reply:
hah sorry I was also commenting on this as you were typing yours up. Closing this conversation in favour of the one below